### PR TITLE
Role and policy again

### PR DESCRIPTION
**Pull request by tthyer**
This is exactly the same as before except let's merge it into master as this got merged into the "cluster" branch the first time and we are abandoning that one.

There is nothing new here, all of this was previously approved.

### CODE REVIEW

**Comment by tthyer** [comment]
@zaro0508 This is failing during the travis validate stage. Specifically, on lint_stack. I've started looking into fixing it but would like some assistance. I think this is breaking at line 37. If I'm reading this shell script correctly, it looks to me like I could break this a good bit during development.

File: config/prod/toil-shared-cluster-role-and-policy.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+template_path: remote/toil-cluster-role-and-policy.yaml
+stack_name: toil-cluster-shared-role
```
Comment (zaro0508) [inline config/prod/toil-shared-cluster-role-and-policy.yaml]: there's a `ClusterName` parameter in templates/toil-cluster-role-and-policy.yaml file.  you need to pass in something for that or alternatively remove the parameter.

**Comment by tthyer** [comment]
@zaro0508 I resolved conflicts with master and added the ClusterName parameter. The validate step still fails in the same place with the same error message.

**Comment by tthyer** [comment]
Closing this PR as it is believed to be corrupted in some way

**Pull request closed without merging**